components/team_general_tab.jsx

```diff
@@ -108,11 +108,25 @@ class GeneralTab extends React.Component {
         let valid = true;
 
         const name = this.state.name.trim();
-        if (name) {
-            state.clientError = '';
-        } else {
+
+        if (!name) {
             state.clientError = Utils.localizeMessage('general_tab.required', 'This field is required');
             valid = false;
+        } else if (name.length < Constants.MIN_TEAMNAME_LENGTH) {
+            state.clientError = (
+                <FormattedMessage
+                    id='create_team.display_name.charLength'
+                    defaultMessage='Team Name must be {min} or more characters up to a maximum of {max}. You can add a longer team description later.'
```

Comment:
Sorry, can you also help add this string to en.json? https://github.com/mattermost/mattermost-webapp/blob/master/i18n/en.json as per the [contribution guidelines](https://docs.mattermost.com/developer/contribution-guide.html#preparing-a-pull-request).

Thanks for helping with the fix!

Reply:
@jasonblais vl do it by evening